types.go

```diff
@@ -14,7 +14,7 @@ func (mm MeasurementsMap) AddWithPrefix(prefix string, m MeasurementsMap) Measur
 	return mm
 }
 
-func (mm MeasurementsMap) AddInnerWithPrefix(prefix string, m MeasurementsMap) MeasurementsMap {
+func (mm MeasurementsMap) AddInnerWithPrefix(prefix string, m interface{}) MeasurementsMap {
```

Comment:
Why do we change the type to `interface{}`?

Reply:
Using `MeasurementsMap` causes cyclic dependency issue.